playwright/test/helpers/urls.ts

```diff
@@ -3,3 +3,4 @@ export const baseUrl = process.env.PLAYWRIGHT_BASE_URL
   : 'http://localhost:3000';
 export const worksUrl = `${baseUrl}/works`;
 export const imagesUrl = `${baseUrl}/images`;
+export const useStageApis = 'USE_STAGE_APIS' in process.env;
```

Comment:
We don't care about the value, just that it exists? e.g. `USE_STAGE_APIS=false`?

Reply:
Yeah, but actually I think I prefer checking for a value - will fix.